ec3730/Data Feeds/WhoisXML/WhoisXml.swift

```diff
@@ -96,8 +96,10 @@ extension WhoisXml: DataFeedService {
 
     static var reputationService: WhoisXMLService = .init(name: "Reputation", description: "Our hosted lookup uses hundreds of parameters to calculate reputation scores.", id: "20")
 
+    static var contactsService: WhoisXMLService = WhoisXmlContactsService(name: "Contacts", description: "Contacts service", id: "")
```

Comment:
ID needs to be filled in. You can find it by looking at the response from `https://user.whoisxmlapi.com/user-service/account-balance?apiKey=API_KEY`